### ISSUE TITLE
New Feature: Touch Lock

### ISSUE THREAD

**Issue by hmronline** [thanksmister/wallpanel-android#13]
The idea is to disable or enable Touchscreen using MQTT commands.
When touchscreen is disabled (or touch is locked), there will be no possible interaction with the WallPanel.
This could be helpful for having a CellPhone or Tablet showing something in a public place (like a doordbell) but you don't wan visitors to be able to interact with your app, just see what we want to show.

**Comment by thanksmister** [comment]
I don't think you can disable touch events, I think you mean disable the android controls such has home and back.   The application needs touch interaction in order to interact with the application, I would not disable touch events.   If you want a kiosk application, there are already some available on Google Play.
